src/java/com/wolfssl/provider/jsse/WolfSSLContext.java

```diff
@@ -85,6 +88,10 @@ private void createCtx() throws WolfSSLException {
                     "Invalid SSL/TLS protocol version");
         }
 
+        ctxAttr = ctxAttr.GetCtxAttributes(method, WolfSSL.getCiphersIana());
```

Comment:
The "method" variable here is a "long" type and represents a native pointer that was returned by one of wolfSSL's method functions (ex: wolfSSLv23_client_method(), etc).  There are a few problems with this approach:

It doesn't make sense to hand this "pointer" to the user of GetCtxAttributes().  I think we should we should move the call to GetCtxAttributes() above the switch(this.currentVersion) statement, and pass the user "this.currentVersion" as the first argument to GetCtxAttributes().  This way, the user can pass back something simpler (TLSv1, TLSv1_1, etc).  We can then use their output in the switch statement to create the correct method.

Moving this above the switch() will also prevent a memory leak from the previous WolfSSL.<method> call.

Reply:
Changed method pointer to TLS_VERSION